docs/developer-docs/latest/update-migration-guides/migration-guides/v4/code/backend/policies.md

```diff
@@ -0,0 +1,125 @@
+---
+title: v4 code migration - Policies - Strapi Developer Docs
+description: Migrate policies from Strapi v3.6.8 to Strapi v4.0.x with step-by-step instructions
+canonicalUrl:  Used by Google to index page, should start with https://docs.strapi.io/ — delete this comment when done [paste final URL here]
+---
+
+<!-- TODO: update SEO -->
+
+# v4 code migration: Updating policies
+
+!!!include(developer-docs/latest/update-migration-guides/migration-guides/v4/snippets/code-migration-intro.md)!!!
+
+::: strapi v3/v4 comparison
+
+In both Strapi v3 and v4, policies handle authorization. Policies can be global or scoped (i.e. applied to a specific API or plugin) and can be applied to REST controller's actions or to GraphQL resolvers.
```

Comment:
> Policies can be global or scoped (i.e. applied to a specific API or plugin)

Not quite, Policies can be stored globally or scoped but regardless of location they can be used anywhere. How they are called is what changes but there is no application limitation.

I'm not sure how to word this though :thinking: Not saying this needs to change much but in its current state, it's a half truth.